### PR TITLE
[AL-2320] E2E Tests

### PR DESCRIPTION
**Pull request by jtsodapop**
yaml file for the e2e pipeline tests

### CODE REVIEW

**Comment by jtsodapop** [comment]
Codefresh pipeline can be found here: [link](https://g.codefresh.io/pipelines/edit/workflow?id=626fef2019fe9232c5f872a6&pipeline=python-sdk-tests&projects=labelbox-python&projectId=626fee99024748bd16a9e34b&rightbar=triggers&advanced=0)

Pipeline successfully completes

File: e2e_tests_codefresh_pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    image: "${{build_sdk_test}}"
+    working_directory: "IMAGE_WORK_DIR"
+    commands:
+      - pytest
+  sendSlack:
+    title: "Alert on failure"
+    type: slack-message-sender
+    arguments:
+      WEBHOOK_URL: "${{slack_webhook_url}}"
+      MESSAGE: "Annotations Import pipelines have failed -- ${{CF_BUILD_URL}}"
```
Comment (cssetian) [inline e2e_tests_codefresh_pipeline.yaml:39]: Should we update this to say 'Python SDK e2e tests have failed'?

File: e2e_tests_codefresh_pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    title: "Python SDK e2e tests"
+    description: "Running Python SDK e2e tests"
+    fail_fast: false
+    stage: "e2e_test"
+    image: "${{build_sdk_test}}"
+    working_directory: "IMAGE_WORK_DIR"
+    commands:
+      - pytest
+  sendSlack:
+    title: "Alert on failure"
```
Comment (cssetian) [inline e2e_tests_codefresh_pipeline.yaml:35]: Maybe for a while we should alert on success too to start to make sure things are running smoothly

File: e2e_tests_codefresh_pipeline.yaml (hunk 1 above)
Comment (jtsodapop) [reply e2e_tests_codefresh_pipeline.yaml]: Updated to reflect status being successful !

File: e2e_tests_codefresh_pipeline.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    title: "Alert on success to pages active learning"
+    type: slack-message-sender
+    arguments:
+      WEBHOOK_URL: "${{slack_webhook_url}}"
+      MESSAGE: "SDK Tests have succeeded-- ${{CF_BUILD_URL}}"
+    when:
+      condition:
+        all:
+          succeeded: e2e_test.result == 'success'
+  sendSlackFailureToE2E:
```
Comment (msokoloff1) [inline e2e_tests_codefresh_pipeline.yaml:44]: I'm not super familiar with codefresh pipelines. I assume that these names are just arbitrary stage names. Is there a reason some are camel case and some are snake case?

**Pull request merged**